build/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ option(TRTIS_ENABLE_CUSTOM "Include Custom backend in server" OFF)
 # Endpoints
 option(TRTIS_ENABLE_HTTP "Include HTTP API in server" ON)
 option(TRTIS_ENABLE_GRPC "Include GRPC API in server" ON)
+option(TRTIS_ENABLE_KFSERVING "Include KFServing HTTP API in server" ON)
```

Comment:
TRTIS_ENABLE_HTTP_V2  "Enable version 2 of the HTTP API in server"
Add a check before line 68 below that raises an error is this is set and TRTIS_ENABLE_HTTP is not enabled. So to get V2 you need to enable both flags.

Also, for now just disallow metrics when V2 is enabled. We could break out a metrics_server.cc but for now don't need to do that.

Reply:
Will do